source/cl/test/UnitCL/source/cl_khr_command_buffer/clCommandNDRangeKernelKHR.cpp

```diff
@@ -72,6 +72,55 @@ TEST_F(CommandNDRangeKernelTest, EmptyKernel) {
   ASSERT_SUCCESS(clReleaseProgram(program));
 }
 
+// Tests the whether we can enqueue an execute a command buffer containing a
+// kernel enqueue created via clCommandNDRangeKernelKHR with global size of 0
+// or a global size of nullptr
+TEST_F(CommandNDRangeKernelTest, GlobalZero) {
```

Comment:
The corresponding `clEnqueueNDRangeKernel` test is called `ZeroNDRange`, can we use the same name here? And the difference here where we also check that the kernel is not executed, can we use that in the `clEnqueueNDRangeKernel` test as well?